SingularityUI/app/components/globalSearch/GlobalSearch.cjsx

```diff
@@ -0,0 +1,52 @@
+React = require 'react'
+
+{ Typeahead } = require 'react-typeahead'
+
+
+GlobalSearch = React.createClass
```

Comment:
Forgot to circle back here -- extending `React.Component` comes with some benefits, such as `displayName` being properly set by default when you try to inspect the page